tests/output/test_plotting.py

```diff
@@ -0,0 +1,169 @@
+from firedrake import *
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+
```

Comment:
It's wonderful that this has tests, given that it's a plotting API. Could you perhaps add tests for `plot`, e.g. that it only works when it should?